2018/python/test_file_generator.py

```diff
@@ -212,26 +212,26 @@ def v607_file(self):
 		print("writing {name}".format(name=name))
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
-	# under construction
-	# def v608_1_file(self):
-	# 	"""Set a ULI to be a random choice of 22 characters or 46 characters"""
-	# 	name = "v608_1.txt"
-	# 	path = self.validity_path
-	# 	ts = self.ts_df.copy()
-	# 	lar = self.lar_df.copy()
-	# 	lar['uli'] = random.choice([lar.lei + "DCM78AVG3FFL1YB5H2BR2EDJKLMNO", lar.lei+"AB"])
-	# 	print("writing {name}".format(name=name))
-	# 	utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
-
-	# def v608_2_file(self):
-	# 	"""Set a NULI to be greater than 22 characters."""
-	# 	name = "v608_2.txt"
-	# 	path = self.validity_path
-	# 	ts = self.ts_df.copy()
-	# 	lar = self.lar_df.copy()
-	# 	lar["uli"] = "DCM78AVG3FFL1YB5H2BR2EDJKLMNO"
-	# 	print("writing {name}".format(name=name))
-	# 	utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+	def v608_1_file(self):
+		"""Set a ULI to be a random choice of 22 characters or 46 characters"""
+		name = "v608_1.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar['uli'] = random.choice([lar.lei + utils.char_string_gen(25), lar.lei])
```

Comment:
for the second item in the random choice list, it might be better to not use the LEI as this could make testing/debugging confusing. Are you able to use a random string for the NULI instead?